Alinoker/Dog-breed-quiz
# Initial setup

This PR adds the initial files, establishes some coding guidelines, creates documentation, PR template and scripts for linting - all in the name of a smoother and more consistent developer experience.

## How to test
1. Ensure you have Node.js installed.
2. Check out to the branch with `git fetch; git checkout chore/initial-setup;`
3. Run the following in your terminal to confirm everything is running as expected:
`npm i; npm run lint;`
4. Right click on `src/index.html` > Open with live server > Click the blue button. Confirm you are seeing the alert pop up.

<img width="932" alt="image" src="https://github.com/Alinoker/Dog/assets/53922624/81952c62-d484-4cbf-a6b4-6f4797e04817">
